Felienne/hedy
# Improve class overview UI

Provide a general summary of your changes in the Title of the PR

**Description**
We improve the class overview UI to better align with the UI of the rest of the website. The table is styled and borders are adjusted to better differentiate between different sections. The header titles are shortened in the *.yaml files to better fit the UI and still contain the same message. The styling of the timestamp is improved through some additional pre-processing on the `isoformat()`.

**Fix for**

This PR fixed #1153 and fixed #1154. Originally, the purpose was to only fix #1154 but as they are very much aligned both are solved at once in this PR.

**How to test**

Make sure you have a teacher's account and navigate to one of your classes and notice the difference UI of the class overview.

**Screenshot**
![Schermafbeelding 2021-11-15 om 13 11 51](https://user-images.githubusercontent.com/38155424/141780034-47de00c5-d9e0-4ec2-94c1-3b5d7f5a7ca1.png)

![Schermafbeelding 2021-11-15 om 13 12 17](https://user-images.githubusercontent.com/38155424/141780057-bdbde5ca-4b35-4a78-9e69-b91e3d566838.png)

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
